packages/filesystem/src/browser/download/file-download-service.ts

```diff
@@ -19,19 +19,16 @@ import URI from '@theia/core/lib/common/uri';
 import { cancelled } from '@theia/core/lib/common/cancellation';
 import { ILogger } from '@theia/core/lib/common/logger';
 import { Endpoint } from '@theia/core/lib/browser/endpoint';
-import { StatusBar, StatusBarAlignment } from '@theia/core/lib/browser/status-bar';
 import { FileSystem } from '../../common/filesystem';
 import { FileDownloadData } from '../../common/download/file-download-data';
 import { Deferred } from '@theia/core/lib/common/promise-util';
 import { MessageService } from '@theia/core/lib/common/message-service';
+import { addClipboardListener } from '@theia/core/lib/browser/widgets';
 
 @injectable()
 export class FileDownloadService {
 
-    protected static PREPARING_DOWNLOAD_ID = 'theia-preparing-download';
-
     protected anchor: HTMLAnchorElement | undefined;
-    protected downloadQueue: number[] = [];
```

Comment:
All symbols with `protected` and `public` (default) visibility are part of the API, if you change it, you have to declare your modifications as a breaking change in the [changelog](https://github.com/theia-ide/theia/blob/master/CHANGELOG.md).